src/lib/utils/index.ts

```diff
@@ -252,7 +252,7 @@ export function normalizePathsInData<X>(x: X, basePath?: string, basePathMask?:
     if (basePath) {
       x_ = replaceEvery(x_, basePath, basePathMask ?? '<dynamic>')
     }
-    x_ = replaceEvery(x_, Path.sep, '/')
+    x_ = replaceEvery(x_, Path.sep, Path.posix.sep)
```

Comment:
I learned about the `posix` and `win32` namespaces. They expose path APIs that are NOT dynamic to the OS.